### PR TITLE
Add server attribute copy to TPv2 ISO generation form

### PR DESCRIPTION
**Pull request by ocket8888**
This PR adds a way to copy information from existing servers to autofill parts of the ISO generation form in TPv2

<hr/>

## Which Traffic Control components are affected by this PR?
- Traffic Portal (experimental v2)

## What is the best way to verify this PR?
Make sure the provided tests pass.

## PR submission checklist
- [x] This PR has tests
- [x] This PR doesn't need documentatio
- [x] This PR doesn't need a CHANGELOG.md entry
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY**

### CODE REVIEW

File: experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.html [hunk 1]
```diff
@@ -13,9 +13,10 @@
 -->
 <mat-card>
 	<form [formGroup]="form" (ngSubmit)="submit($event)">
-		<mat-card-header>
+		<mat-card-actions align="start">
+			<button mat-raised-button type="button" (click)="openCopyDialog()">Copy Info From Server</button>
```
Comment (shamrickus) [inline experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.html:17]: I think this button should be right aligned (Use DHCP still on left though).

File: experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.html [hunk 2]
```diff
@@ -13,9 +13,10 @@
 -->
 <mat-card>
 	<form [formGroup]="form" (ngSubmit)="submit($event)">
-		<mat-card-header>
+		<mat-card-actions align="start">
```
Comment (shamrickus) [inline experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.html:16]: This should still be `mat-card-header`, The [docs](https://material.angular.io/components/card/overview) define `mat-card-actions` as being bottom aligned and `mat-card-header` as being top aligned.

**Review by shamrickus — APPROVED** [review]
LGTM
Just needs a rebase

File: experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.spec.ts [hunk 3]
```diff
@@ -49,6 +88,46 @@ describe("ISOGenerationFormComponent", () => {
...
+		}
+
+		server = await serverSrv.createServer({
+			cachegroupId: cg.id,
+			cdnId: cdn.id,
+			domainName: "test",
+			hostName: "quest",
+			interfaces: [iface],
+			physLocationId: physLoc.id,
+			profileId: profile.id,
```
[the remark this answers is not in the document]
Comment (ocket8888) [reply experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.spec.ts:127]: huh. idk why that didn't fail to build.

[the remark this answers is not in the document]
Comment (ocket8888) [reply experimental/traffic-portal/src/app/core/misc/isogeneration-form/isogeneration-form.component.spec.ts:127]: oh, nvm, it did, but that's a spec file so it wasn't in the compilation unit for the other GHA's

**Pull request merged**